cli/cmd/install.go

```diff
@@ -359,6 +362,12 @@ control plane.`,
 				os.Exit(1)
 			}
 
+			shouldExist := options.identityOptions.identityExternalIssuer
+			if err := errIfLinkerdIdentityissuerSecretMissing(shouldExist); err != nil && !options.ignoreCluster {
```

Comment:
You could wrap that condition with `if !options.ignoreCluster && shouldExist` and then have `errIfLinkerdIdentityissuerSecretMissing()` have no args and only check when `shouldExist` is true, to avoid the network calls when `shouldExist` is false.

Reply:
As a matter of fact, I'm unable to run a naked `linkerd install` because it's forcing me to have the namespace already. This is probably also why the integration tests are failing.